fhir-persistence-schema/src/main/java/com/ibm/fhir/schema/app/util/SchemaSupport.java

```diff
@@ -59,10 +59,7 @@ public String getResourceTypeFromTableName(String tableName) {
             int idx = tableName.indexOf('_');
             if (idx > 0) {
                 String sub = tableName.substring(0, idx);
-                resourceType = ALL_RESOURCE_TYPES.get(sub.toUpperCase());
-                if (resourceType == null) {   
-                    throw new IllegalArgumentException("Not a recognized data schema table name: " + tableName);
-                }
+                resourceType = ALL_RESOURCE_TYPES.get(sub.toUpperCase()); // may be null
```

Comment:
not sure what are the ramifications of allowing null through here, but I assume you've considered that

Reply:
Yes, described in the javadoc:
```
     * @return the associated resource type or null if the table is known but not associated with a resource
```